src/main.js

```diff
@@ -226,9 +226,46 @@ define(['storage'], function(storage) {
     gmi.gameLoaded();
     gmi.sendStatsEvent('game_loaded', true, {});
 
+    // ---------- Web Sockets Example ----------
```

Comment:
Should all of this be inside `main.js`, might it be better in `startup.js` (or in a separate file called by `startup.js`)?